info/guide/diktat-coding-convention.md

```diff
@@ -1877,6 +1877,30 @@ fun foo() {
     println("Nested Output: ${nested()}") 
 } 
 ```  
+#### <a name="r5.1.4"></a> 5.1.4 Negated function calls
```

Comment:
You should add this to chapter 5 first, not in the merged document. Also please add it to TOC

Reply:
Fixed